service-api/app/test/AppTest/DataAccess/DynamoDb/ViewerCodesTest.php

```diff
@@ -85,10 +90,13 @@ public function cannot_lookup_a_missing_code(): void
                 $this->assertEquals(['S' => $testCode], $data['Key']['ViewerCode']);
 
                 return true;
-            }))
-            ->willReturn($this->createAWSResult([
-                'Item' => []
-            ]));
+            })
+        )
+            ->willReturn(
+                $this->createAWSResult([
+                                           'Item' => [],
+                                       ])
```

Comment:
```suggestion
                $this->createAWSResult(
                    [
                        'Item' => [],
                    ]
                )
```
Something like this.